src/card/block.spec.js

```diff
@@ -0,0 +1,180 @@
+            await page.focus( '.block-editor-rich-text__editable.content' );
+            await page.keyboard.type(typeString, {delay: 10})
+
+            const editedContent = await getEditedPostContent()
+
+            // tests: 
+            // a. save output properly contains the correct string 
+            // b. save output matches the existing snapshot
+            expect( editedContent.includes(`<p class="content">${typeString}</p>`)).toBe(true)
+            expect( editedContent).toMatchSnapshot()
+        })
+    })
+    describe( '🔬 Side Panel Settings', () => {
+        test( '🔬 Gold background color should be selectable.', async () => {
+            await blockStartup(block)
+
+            // select the radio button with the virtual keyboard
+            await clickRadio('Background Color', 'gold')
+
+            const editedContent = await getEditedPostContent()
+
+            expect( editedContent.includes(`"backgroundColor":"white"`)).toBe(false)
+            expect( editedContent).toMatchSnapshot()
+        })
+        test( '🔬 Black border color should be selectable.', async () => {
+            await blockStartup(block)
+        
+            // select the radio button with the virtual keyboard
+            await clickRadio('Border Color', 'black')
+
+            const editedContent = await getEditedPostContent()
+
+            expect( editedContent.includes(`"borderColor":"gold"`)).toBe(false)
+            expect( editedContent).toMatchSnapshot()
+        })
+        test( '🔬 Image alt text should be editable.', async () => {
+            await blockStartup(block)
+
+            const typeString = "Image alt text."
+            
+            // focus the text box then type into it with the virtual keyboard
+            await clickElementByText('label', 'Image Alt Text')
+            await page.keyboard.type(typeString, {delay: 10})
+
+            const editedContent = await getEditedPostContent()
+
+            // tests: 
+            // a. save output properly contains the correct string 
+            // b. save output matches the existing snapshot
+            expect( editedContent.includes(`"altText":"${typeString}"`)).toBe(true)
+            // expect( editedContent).toMatchSnapshot()
+        })
+        describe( '🔬 Link can be added to the card', () => {
+            test( '🔬 "Add a link to this card?" checkbox reveals link inputs', async () => {
+                await blockStartup(block)
+
+                await clickCheckbox('Add a link to this card?')
+        
+                const linkInputs = await page.$x(
+                    `//label[@class = "components-checkbox-control__label"][.="Call to action text" or .="Link address" or .="Open link in new tab?"]`
+                )
+        
+                // check that the checkboxes appear and that the correct number exist, 3
+                expect(linkInputs).not.toBeNull()
+                expect(linkInputs.length).toBe(1)
```

Comment:
Shouldn't this be testing for three items in the array? 

`expect(linkInputs.length).toBe(3)`

Reply:
It was a mistake. Updated it in a new commit.